docs/guides/opera_rtc_product_guide.md

```diff
@@ -0,0 +1,512 @@
+
+  - In Vertex, search for both `RTC` and `RTC-STATIC` file types for your area of interest. 
+  - Because the static layers all have an acquisition date set to `04/03/2014, 00:00:00Z`, the static 
+    files may not be included in the results if a date range has been applied to the search. 
+  - Select an RTC product from the results list that has the desired footprint, and click on the icon button
+    next to the Opera Burst ID.
+  - Select the option to `Add OPERA Burst ID to Search` and click the Search button again.
+  - Make sure to clear any date range filters that do not include April 3, 2014.
+  - The search results will include only the products that have the designated burst ID. If there are too many 
+    results for the RTC-STATIC products to be included in the search results, open the Search Filters panel again and 
+    select only the RTC-STATIC product type to return only the static layers that correspond to that RTC footprint.
+
+### Duplicate layer names
+
+There are **Local Incidence Angle** and **Mask** files listed in association with both the `RTC` search results 
+and the `RTC-STATIC` search results. 
+
+#### Local Incidence Angle
+
+The Local Incidence Angle file can be downloaded using either link, but it is the same file. It is always named with 
+this pattern: 
+`OPERA_L2_RTC-S1-STATIC_Txxx-xxxxxx-IWx_20140403_S1A_30_v1.0_local_incidence_angle.tif`
+
+The download URL behind the local incidence angle listings in both the `RTC` and `RTC-STATIC` results 
+reference the same source file.
+
+#### Validity Mask
+
+There is a file called `Mask` listed with both the `RTC` and the `RTC-STATIC` search results. Unlike the 
+local incidence angle file, these two mask files are NOT the same. Both are validity masks with the same pixel 
+value categories: 
+
+| Pixel Value | Description                                    |
+|-------------|------------------------------------------------|
+| 0           | Valid sample not affected by layover or shadow |
+| 1           | Valid sample affected by shadow                |
+| 2           | Valid sample affected by layover               |
+| 3           | Valid sample affected by layover and shadow    |
+| 255         | Invalid sample (fill value)                    |
+
+The main difference between the two files is that the data extent matches the other associated files. The 
+`RTC-STATIC` files have a larger extent than the `RTC` files, as data is included for the full raster footprint, 
+including the NoData pixels that are present around the edges of the RTC data products.
+
+![Figure 5](../images/opera-mask-compare.png "Comparison of the two OPERA RTC-S1 validity masks")
+
+The mask file in the map on the left is the mask linked to the `RTC` search results.
+
+- `OPERA_L2_RTC-S1_T115-245714-IW1_20250418T141628Z_20250419T010229Z_S1A_30_v1.0_mask.tif`
+
+The extent of NoData padding around area that has valid radar data for that particular burst is displayed with a 
```

Comment:
```suggestion
The extent of NoData padding around the area that has valid radar data for that particular burst is displayed with a 
```

Reply:
Removing 'the' sounds fundamentally wrong to me.

Reply:
@hjkristenson I think originally 'the' was missing

Reply:
Ah, I see! Yep, should definitely be there. Thanks!